velox/common/encode/Base64.h

```diff
@@ -110,6 +110,11 @@ class Base64 {
       char* outputBuffer,
       size_t outputSize);
 
+  /// Encodes the input buffer into Base64 MIME format.
+  /// Inserts a CRLF every kMaxLineLength output characters.
+  static void
+  encodeMime(const char* input, size_t inputSize, char* outputBuffer);
```

Comment:
Do we have a reference of implementation for the encoding process? Perhaps provide it in the comment.